packages/dd-trace/src/tracer.js

```diff
@@ -77,7 +77,7 @@ class DatadogTracer extends Tracer {
           }
         )
       } else {
-        span.finish()
+        span.finish(options.finishTime)
```

Comment:
This is interesting! What's happening here?

Reply:
`span.finish()` takes a finishTime argument but it wasn't getting passed down in the `tracer.startSpan` function call.  This is needed for the `prisma.engine` spans as the engine binray returns an array of span attributes with a start and stop time that are then built into spans.

Reply:
Removed this field as it was not needed